assets/js/blocks/rating-filter/preview.tsx

```diff
@@ -61,3 +61,26 @@ export const previewOptions = [
 		value: '1',
 	},
 ];
+
+export const editorPreviewOptions = [
+	{
+		rating: 5,
+		count: 1,
+	},
+	{
+		rating: 4,
+		count: 1,
+	},
+	{
+		rating: 3,
+		count: 1,
+	},
+	{
+		rating: 2,
+		count: 1,
+	},
+	{
+		rating: 1,
+		count: 1,
+	},
+];
```

Comment:
Instead of creating a new variable, can we reuse the `previewOptions` above?

Reply:
Yes, we can: I initially kept them separated in case we wanted to display different content on preview mode x the editor, but since at this point they don't differ, I've adopted your suggestion here and consolidated both 👍